lib/util/Util.js

```diff
@@ -108,7 +108,7 @@ module.exports = class Util {
             accumulator[key] = val.replace(/"/g, "") ;
             return accumulator;
         };
-        const intentRegex = /([\w|.]*)((?: \w*=(?:"(?:[^"])*"|(?:[^ ])*))*)/;
+        const intentRegex = /([\w|.|-|_]*)((?: \w*=(?:"(?:[^"])*"|(?:[^ ])*))*)/;
```

Comment:
Probably a test with these characters is needed

Reply:
A test with the relevant characters has been added